packages/cli/src/util/__tests__/colorizer.spec.ts

```diff
@@ -0,0 +1,135 @@
+import chalk from 'chalk';
+import {
+  PRE_PARAM_VALUE_TAG,
+  POST_PARAM_VALUE_TAG,
+  attachTagsToParamsValues,
+  transformPathParamsValues
+} from '../colorizer';
+
+import { createPrismServerWithLogger } from '../createServer';
+import { PassThrough } from 'stream';
+import { createLogger } from '@stoplight/prism-core';
+import * as httpServer from '@stoplight/prism-http-server';
+
+const logStream = new PassThrough();
+const logInstance = createLogger('CLI', undefined, logStream);
+
+describe('colorizer', () => {
+
+  if (chalk.supportsColor) {
+    it('colorizes values for parameters', async () => {
+      // @ts-ignore
+      jest.spyOn(httpServer, 'createServer').mockImplementationOnce(() => {
+        return {
+          listen: () => {}
+        }
+      });
+
+      jest.spyOn(logInstance, 'note').mockImplementation((aString) => {
+        // @ts-ignore
+        const coloredParam = aString.match(/(endpoint\/)(.*)/)[2];
+
+        expect(coloredParam).toContain("\u001b[1m\u001b[36m");
+        expect(coloredParam).toContain("\u001b[39m\u001b[22m");
+      });
+
+      await createPrismServerWithLogger({
```

Comment:
…and return instead of awaiting.

```suggestion
      return createPrismServerWithLogger({
```